ncwb-activity-recorder/src/Record.Save.Service.js

```diff
@@ -0,0 +1,89 @@
+import { flushEvents, getEvents } from "./Record.RRweb-Record.Service";
+
+function _getNcwbEl() {
+  let ncwbScript = document.querySelector('script[data-ncwb-id="ncwb"]');
+  return ncwbScript;
+}
+
+function _getUserId() {
+  return _getNcwbEl().getAttribute('data-user-id');
+}
+
+function _getWebsiteId() {
+  return _getNcwbEl().getAttribute('data-website-id');
+}
+
+let visitorActivityId = null;
+function _setVisitorActivityId(id) {
+  visitorActivityId =id;
+}
+
+function _fetch(payload, url, method) {
+  fetch(url, {
+    method,
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+  }).then((res=> {
+    flushEvents();
+    _setVisitorActivityId(res.id);
+  }));
+}
+
+function _getSavePayload(rating) {
+  const payload = {
+    activityType: 'FEEDBACK',
+    userId: _getUserId(),
+    websiteId: _getWebsiteId(),
+    activityEvents: getEvents(),
```

Comment:
Break this function into error save and feebdack save services with commonlyy used functions located in record.service.js